templates/firefox-ios.xliff

```diff
@@ -2008,6 +2032,14 @@
         <source>Clear Private Data</source>
         <note>Title displayed in header of the setting panel.</note>
       </trans-unit>
+      <trans-unit id="Settings.ClearSelectedWebsiteData.ClearOneItem.Button" xml:space="preserve">
+        <source>Clear 1 Item</source>
+        <note>Button in Data Management that clears private data for one selected item.</note>
+      </trans-unit>
+      <trans-unit id="Settings.ClearSelectedWebsiteData.ClearSelected.Button" xml:space="preserve">
+        <source>Clear %1$@ Items</source>
+        <note>Button in Data Management that clears private data for the selected items. Parameter is the number of items to be cleared</note>
```

Comment:
I think we still don't correctly support plurals.
Workaround would be: Clear Items: %1$@
(and even if it's zero, not a big deal)